services/storage-hal/Dockerfile

```
@@ -0,0 +1,55 @@
+
+# Expose ports
+EXPOSE 50051 9090
+
+# Set environment variables
+ENV RUST_LOG=info
+ENV SERVICE_PORT=50051
+ENV METRICS_PORT=9090
+
+# Health check
+HEALTHCHECK --interval=30s --timeout=3s --start-period=5s --retries=3 \
+    CMD [ "/usr/local/bin/storage-hal", "--health-check" ] || exit 1
```

Hadolint (2.14.0)

[error] unexpected '|'
expecting '\', a new line followed by the next instruction, or at least one space

(DL1000)